core/new-gui/src/app/workspace/component/left-panel/left-panel.component.ts

```diff
@@ -25,6 +26,7 @@ export class LeftPanelComponent implements OnDestroy, OnInit {
     { component: null, title: "", icon: "", enabled: true },
     { component: OperatorMenuComponent, title: "Operators", icon: "appstore", enabled: true },
     { component: VersionsListComponent, title: "Versions", icon: "schedule", enabled: environment.userSystemEnabled },
+    { component: WorkflowExecutionModalComponent, title: "Execution History", icon: "history", enabled: true },
```

Comment:
We want to link enabled with Workflow Execution flag.

Reply:
fixed